ui/src/types/alerting.ts

```diff
@@ -1,9 +1,58 @@
+import {StatusRule, NotificationRuleBase, TagRule} from 'src/client'
+
+export interface AddID<T> {
+  id: string
+  value: T
+}
+
+export type StatusRuleItem = AddID<StatusRule>
```

Comment:
Continuing the earlier train of thought, maybe these could be `DraftStatusRule` and `DraftTagRule`, which would both be part of a `DraftNotificationRule`.

Reply:
Yup yup